exist-versioning-release.md

```diff
@@ -140,11 +140,75 @@ Once development on a new stable version is complete, the following steps will p
+        ```
+        $ keytool -genkeypair --alias exist-release-build -storepass exist-release-build-password -validity 9999 -keystore /home/my-username/exist-release-build_key.store
+        ```
+
+    4. Perform the build of the tag:
+        ```
+        $ git checkout eXist-3.1.0
+        $ ./build.sh jnlp-unsign-all all jnlp-sign-exist jnlp-sign-core
+        $ ./build.sh installer app-signed dist-war
+        ```
+
+#### Publishing the Product Release
+
+1. Login to https://bintray.com/existdb/ and create a new "Version", then upload the files `$EXIST_HOME/installer/eXist-db-setup-3.2.0.jar`, `$EXIST_HOME/dist/eXist-db-3.2.0.dmg` and `$EXIST_HOME/dist/exist-3.1.0.war`. Once the files have uploaded, make sure to click "Publish" to publish them to the version. Once published, you need to go to the "Files" section of the version, and click "Actions"->"Show in downloads list" for each file.
+
+2. Update and publish the latest Maven artifacts as described here: https://github.com/exist-db/mvn-repo
+
+3. Edit the links for the downloads on the eXist website, by logging into eXide on http://www.exist-db.org/exist/apps/eXide/ and opening the file `/db/apps/homepage/index.html`, you need to modify the HTML under `<a name="downloads"/>` and then save the page:
+
+   ```html
+   <a name="downloads"/>
+   <div class="row">
+     <div class="col-md-12">
+         <h2 id="download">Download</h2>
+         <a href="https://bintray.com/existdb/releases/exist/3.1.0/view">
+             <button class="btn btn-default download-btn stable" type="button">
+                 <span class="status">Latest Release</span>
+                 <span class="icon">
+                     <i class="fa fa-download"/>
+                 </span>
+                 <span class="exist-version">Version 3.1.0</span>
+             </button>
+         </a>
+         <a href="https://github.com/exist-db/mvn-repo">
+             <button class="btn btn-default download-btn maven" type="button">
+                 <span class="status">Maven Artifacts</span>
+                 <span class="icon">
+                     <i class="fa fa-github"/>
+                 </span>
+                 <span class="exist-version">Version 3.1.0</span>
+             </button>
+         </a>
+   ```
+
+    As a temporary measure, you also need to make the same modifications to the file in GitHub - https://github.com/eXist-db/website/blob/master/index.html and send a PR for that. 
+
+4. Login to the blog at http://exist-db.org/exist/apps/wiki/blogs/eXist/ and add a new news item which announces the release and holds the release notes. It should be named like http://exist-db.org/exist/apps/wiki/blogs/eXist/eXistdb310
+
+5. Visit the GitHub releases page https://github.com/eXist-db/exist/releases and create a new release, enter the tag you previously created and link the release notes from the blog and the binaries from BinTray.
```

Comment:
Consider adding this after number 5:

5. (b) Update Homebrew's cask (Homebrew is a package manager for macOS):

    ```
    # install and setup script - only needed once
    brew install vitorgalvao/tiny-scripts/cask-repair
    cask-repair --help

    # use to update exist-db
    cask-repair exist-db
    ```

    See [Updating a Cask](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask) for more information. The `cask-repair` command prompts you for the version number of the eXist release (e.g., you enter `3.2.0`), and it will fetch the new version's installer from bintray, calculate its sha-256 hash, and submit a PR on your behalf to the Homebrew Caskroom project. Usually the PR is merged within a few hours.